plugins/ks.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ks
+spec:
+  version: v0.0.43
+  homepage: https://github.com/kubesphere-sigs/ks
+  shortDescription: Simple management of KubeSphere components
+  description: |
+    This plugin simplifies management of KubeSphere components. KubeSphere helps
+    to run continuous delivery pipelines on Kubernetes. You can enable/disable,
+    inspect logs, or upgrade KubeSphere components.
+  caveats: |
+    ks is now installed! To start using kubectl plugins, you need to install
```

Comment:
I'm not sure the "ks is now installed" wording should be here. technically the caveats will be displayed if someone did `kubectl krew info ks`. for example
```
$ kubectl krew info ctx
NAME: ctx
INDEX: default
URI: https://github.com/ahmetb/kubectx/archive/v0.9.4.tar.gz
SHA256: 91e6b2e0501bc581f006322d621adad928ea3bd3d8df6612334804b93efd258c
VERSION: v0.9.4
HOMEPAGE: https://github.com/ahmetb/kubectx
DESCRIPTION:
Also known as "kubectx", a utility to switch between context entries in
your kubeconfig file efficiently.

CAVEATS:
\
 | If fzf is installed on your machine, you can interactively choose
 | between the entries using the arrow keys, or by fuzzy searching
 | as you type.
 | See https://github.com/ahmetb/kubectx for customization and details.
/
```

Reply:
Faire enough. So if it's ok to remove `ks is now installed!` directly?

Reply:
ya I think that should be removed. I'm also confused about the "kubectl plugins" here. did you mean something else?

> To start using kubectl plugins, you need to install
> KubeSphere first:
> $ kk create cluster --with-kubesphere [version]
>    You can find documentation at
>    https://github.com/kubesphere/kubekey

I'm guessing you meant something like `To start using this plugin...` instead of `To start using kubectl plugins...`? if that's the case then I think including a link to kubesphere installation docs would be enough instead of including this other `kk` binary. I personally think something like this would be a little more clear:
```
You need to have KubeSphere installed in your cluster to use this plugin.
See https://github.com/kubesphere/kubesphere#installation for installation instructions
```

if I've misunderstood the intent of what you had in the caveats please let me know!

Reply:
It is much clear. You got it 100%.